src/Native/MatrixFactorizationNative/UnmanagedMemory.cpp

```diff
@@ -9,25 +9,48 @@
 
 using namespace mf;
 
+EXPORT_API(mf_parameter) make_param(const mf_parameter_bridge *param_bridge)
+{
+    mf_parameter param;
+    param.fun = param_bridge->fun;
+    param.k = param_bridge->k;
+    param.nr_threads = param_bridge->nr_threads;
+    param.nr_bins = param_bridge->nr_bins;
+    param.nr_iters = param_bridge->nr_iters;
+    param.lambda_p1 = param_bridge->lambda_p1;
+    param.lambda_p2 = param_bridge->lambda_p2;
+    param.lambda_q1 = param_bridge->lambda_q1;
+    param.lambda_q2 = param_bridge->lambda_q2;
+    param.eta = param_bridge->eta;
+    param.alpha = param_bridge->alpha;
+    param.c = param_bridge->c;
+    param.do_nmf = static_cast<bool>(param_bridge->do_nmf);
```

Comment:
Instead of static cast, would it be better to use `param_bridge->do_nmf != 0 ? true : false`?

Reply:
Will do `> 0 ? true : false`. Thanks.